src/BlockTypes/ClassicTemplate.php

```diff
@@ -35,7 +36,23 @@ protected function initialize() {
 		parent::initialize();
 		add_filter( 'render_block', array( $this, 'add_alignment_class_to_wrapper' ), 10, 2 );
 		add_filter( 'woocommerce_product_query_meta_query', array( $this, 'filter_products_by_stock' ) );
+		add_action( 'enqueue_block_assets', array( $this, 'enqueue_block_assets' ) );
```

Comment:
This filter might not work without gutenberg. it is responsible for ensuring the frontend styles apply in the editor views. @wavvves

Reply:
Requires https://github.com/WordPress/gutenberg/pull/49655 It doesn't look like we can enqueue other frontend styles without this patch because the stylesheets don't contain `wp-block-` rules. Probably needs to be a #wontfix item.